### PR TITLE
Piwik Analytics

### PR DESCRIPTION
**Pull request by t3chguy**
See vector-im/riot-web/pull/4056

### CODE REVIEW

File: src/Analytics.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        this.tracker = null;
+    }
+
+    set(tracker) {
+        this.tracker = tracker;
+
+        this.tracker.enableHeartBeatTimer();
+        this.tracker.enableLinkTracking(true);
+
+        // dis.register(this._onAction.bind(this));
```
Comment (ara4n) [inline src/Analytics.js]: tracking the full dispatcher feels very dubious - i assume this was an experiment that can be removed? :)

Comment (t3chguy) [inline src/Analytics.js]: that's exactly what it was :P Needed a bunch of data points quickly haha

**Comment by t3chguy** [comment]
Note, de-duplicating router explicitly on first page
image

**Comment by ara4n** [comment]
lgtm

**Pull request merged**